EPA/professional_discussion.md

```diff
@@ -0,0 +1,122 @@
+# Professional Discussion
+
+* Placeholder for Table Of Contents - Don't remove
+{:toc}
+
+## Summary
+The Professional Discussion is a one-hour interview with an independent assessor who will ask pre-prepared open questions to test your understanding of the criteria below. It is designed to give you the opportunity to demonstrate knowledge that is difficult to cover or to demonstrate in your project.
+

```

Comment:
Should we link to the Google doc too? It's nice to have a way to view the KSBs too, and there are more questions in there. I guess maybe not if this is public and just leave a link in the PDF?

Reply:
I don't see any issue in sharing; I'll do that